XWalkView/XWalkView/XWalkStubGenerator.swift

```diff
@@ -69,3 +69,41 @@ class XWalkStubGenerator {
         return "function(\(list)) {\n    \(body)\n}"
     }
 }
+
+private extension NSNumber {
+    var isBool: Bool {
+        get {
+            let trueNumber = NSNumber(bool: true)
+            let falseNumber = NSNumber(bool: false)
+            if (self.isEqualToNumber(trueNumber) && self.objCType == trueNumber.objCType)
+            || (self.isEqualToNumber(falseNumber) && self.objCType == falseNumber.objCType) {
+                return true
+            }
+            return false
+        }
+    }
+}
+
+private func toJSONLiteral(object: AnyObject, isPretty: Bool=false) -> String {
+    switch object {
+    case is NSNull:
+        return "null"
+    case is NSError:
+        return "\(object)"
+    case let number as NSNumber:
+        if number.isBool {
+            return (number as Bool).description
+        } else {
+            return (number as NSNumber).stringValue
+        }
+    case is NSString:
+        return object as String
+    default:
+        if let data = NSJSONSerialization.dataWithJSONObject(object,
+            options: isPretty ? NSJSONWritingOptions.PrettyPrinted : nil,
+            error: nil) as NSData? {
+                return NSString(data: data, encoding: NSUTF8StringEncoding) ?? ""
+        }
+        return ""
```

Comment:
Actually I think throwing an exception is not a good idea, as Swift does not provide try/catch and using try/catch is not encouraged in objc. I suppose we'd better print out an error message here, which would be useful enough for debugging